examples/sentinel/README.md

```diff
@@ -0,0 +1,101 @@
+# Sentinel Example
+
+This example shows how to write cost policies with Hashicorp's [Sentinel](https://www.hashicorp.com/sentinel).  For simplicity, this example evaluates policies with the Sentinel CLI (a.k.a. Sentinel Simulator).
```

Comment:
```suggestion
This example shows how to write cost policies with Hashicorp's [Sentinel](https://www.hashicorp.com/sentinel). For simplicity, this example evaluates policies with the Sentinel CLI (a.k.a. Sentinel Simulator). The point of this example is to show how a policy could be written against the Infracost JSON format, not how to run Sentinel, since that's tied to HashiCorp's cloud platform.
```